doc/source/serve/deployment-graph-cookbook.md

```diff
@@ -0,0 +1,60 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.6
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(serve-deployment-graph-cookbook)=
+
+# Deployment Graph Cookbook
+
```

Comment:
can we have a short intro here to show what this doc aims to do ? Also with guided links to quickly jump forward to every section.